wireless_modules_py/sensors/wind_sensor.py

```diff
@@ -0,0 +1,77 @@
+from sensor_base import Sensor
+import serial
+import time
+
+
+class WindSensor(Sensor):
+    """
+    A class for an anemometer connected via serial port.
```

Comment:
Add more documentation here, include the attributes of this class, this reduces the need for redundant comments in your code.

Reply:
By attributes of this class, did you mean variables like baudrate, sensor, readings etc.? If so, I'm just wondering if it's necessary to document them in the docstring for the class? Since all these attributes can be easily found inside the init method and their names are pretty self-explanatory?

Reply:
Yeah you're right, you can disregard this suggestion.